src/sonic-config-engine/portconfig.py

```diff
@@ -263,7 +263,7 @@ def parse_platform_json_file(hwsku_json_file, platform_json_file):
 
 def get_breakout_mode(hwsku=None, platform=None, port_config_file=None):
     if not port_config_file:
-        port_config_file = device_info.get_path_to_port_config_file()
+        port_config_file = device_info.get_path_to_port_config_file(hwsku)
```

Comment:
You defined the order of the parameters as `get_path_to_port_config_file(asic=None, hwsku=None)`, this `hwsku` parameter will be passed in as `asic`

Reply:
fixed, changed the order '''get_path_to_port_config_file(hwsku=None, asic = None)'''